cypress/integration/computation_menu.spec.ts

```diff
@@ -0,0 +1,116 @@
+const years = [
+    2017,
+    2013,
+    2009,
+    2005,
+    2001,
+    1997,
+    1993,
+    1989,
+    1985,
+    1981,
+    1977,
+    1973,
+    1969,
+    1965,
+    1961,
+    1957,
+    1953,
+    1949,
+    1945,
+];
+const yearsWithModernElectionSystem = [2017, 2013, 2009, 2005];
+
+const arbitraryNumberOfMilliseconds = 4000;
+describe("ComputationMenu", () => {
+    beforeEach(() => {
+        cy.clearLocalStorage();
+    });
+    context("default settings", () => {
+        it("work for all years", () => {
+            cy.visit("").wait(arbitraryNumberOfMilliseconds);
+
+            years.forEach((year) => {
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+            });
+        });
+    });
+
+    context("merge to 11 districts checkbox", () => {
+        it("works for relevant years", () => {
+            cy.visit("").wait(1000);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)